en/plugins.rst

```diff
@@ -442,6 +442,9 @@ You can use ``TableRegistry`` to load your plugin tables using the familiar
 
     $contacts = TableRegistry::get('ContactManager.Contacts');
 
+Alternatively you can use:
```

Comment:
Also, shell etc.
Basically anywhere where you got the ModelAware trait attached afaik.